flashcards/readme.md

```diff
@@ -236,3 +236,19 @@ For this reason suggested approach would be to install cypress locally and run i
 
 2. To remove test data from database run:
     > `docker compose exec <backend_service_name> python manage.py delete_test_data`
+
+
+## Using Json Server
+1. To start json-server run:
```

Comment:
Start it automatically during startup of dev application

Reply:
Json server can't start automatically, because it will be used only when backend isn't ready

Reply:
Okay, but json server runs on different port anyway, so if you're gonna use json server to get api data, you're gonna have to use a 3001 port anyway.
On the second note, api uses `/api/v1/<path>` and according to docs json server will put data at `/<path>`

Reply:
@pawelkonior Could you look into it? You said that it shouldn't start with dev, but what Krzysiek says also makes sense.